content/actions/publishing-packages/publishing-nodejs-packages.md

```diff
@@ -93,6 +93,8 @@ registry=https://registry.npmjs.org/
 always-auth=true
 ```
 
+Please note that you need to configure the `registry-url` in `setup-node` to properly configure your credentials.
```

Comment:
Does this also apply to `registry-url: 'https://npm.pkg.github.com'` in the GitHub packages workflow below?

Reply:
the registry URL is a consistent thing that should not be changed as changing it would mean no longer publishing to the registries as oulined in the example, it should not be customized.

`https://registry.npmjs.org/` is actually the default in the npm CLI and the only reason we need to be explicit with it here is due to an implementation detail of `setup-node`. If we manage to fix that specific behavior in `setup-node` we can simplify the example to not specify the registry URl

Reply:
Would this be accurate then? On first read of the current text, I though the user needed to provide a custom value:
```suggestion
Please note that you need to set the `registry-url` to `https://registry.npmjs.org/` in `setup-node` to properly configure your credentials.
```

Reply:
https://apps.apple.com/sa/story/id1538632801?l=ar